check_file_ages_in_dirs

```diff
@@ -46,6 +46,8 @@ The following arguments are accepted:
   -t              (Optional) The time unit used for the -w and -c values.  Must
                   be one of: seconds, minutes, hours, days.  Defaults to hours.
 
+  -I              (Optional) A space separated list of files to ignore.
```

Comment:
These have to be enclosed in quotes, no? Perhaps that would be worth mentioning in the help?